File: src/Sensor.java
public class Sensor {
	
	private int id;
	private String name;
	private int gatewayId;
	private double latency;
	
	public Sensor(int id, String name, int gatewayId, double latency) {
		this.id = id;
		this.name = name;
		this.gatewayId = gatewayId;		
		this.latency = latency;
	}
	
	public void sendData(FogDevice Gateway, String data) throws Exception {		
		Gateway.doTask(data, this.latency);
	}
	
	public double getLatency() {
		return this.latency;
	}
	

}

File: sensor.py
class Sensor:
    def __init__(self, id, name, latency, priority):
        self.id = id
        self.name = name
        self.latency = latency
        self.priority = priority

    def sendData(self, lb, priority, latency, data, fogs):
        # print('sending...')
        lb.allocatedTask(priority, latency, data, fogs)
        # Gateway.doTask(data, latency, data)

    def getLatency(self):
        return self.latency
    def getPriority(self):
        return self.priority
    def getId(self):
        return self.id
